resources/js/views/App.vue

```diff
@@ -33,7 +33,18 @@
 
         <b-collapse id="nav-collapse" is-nav>
           <b-navbar-nav>
-            <b-nav-item :to="{ name: 'rooms.index' }" v-if='isAuthenticated'>{{ $t('rooms.rooms') }}</b-nav-item>
+            <b-nav-item :to="{ name: 'rooms.own_index' }" v-if='isAuthenticated'>{{ $t('rooms.myRooms') }}</b-nav-item>
+            <b-nav-item :to="{ name: 'rooms.index' }" v-if='isAuthenticated'>
+              <can method='viewAll' policy='RoomPolicy'>
+                {{ $t('rooms.allRooms') }}
+              </can>
+              <cannot method='viewAll' policy='RoomPolicy'>
+                {{ $t('rooms.findRooms') }}
+              </cannot>
+            </b-nav-item>
+            <can v-if='isAuthenticated' method='viewAny' policy='MeetingPolicy'>
```

Comment:
This or the settings navigation leads to errors on logout. I think the v-if='isAuthenticated' cannot be used at the can/cannot components